src/main/java/dynamok/sink/DynamoDbSinkTask.java

```diff
@@ -89,14 +89,25 @@ public void start(Map<String, String> props) {
         remainingRetries = config.maxRetries;
     }
 
+    private void attemptPutItem(SinkRecord record) {
+        try {
+            client.putItem(tableName(record), toPutRequest(record).getItem());
+        } catch (AmazonDynamoDBException e) {
+            if(e.getErrorCode().equals("ValidationError")
+                    && e.getErrorMessage().contains("Item size has exceeded the maximum allowed size")) {
+                log.error("Caught item size has exceeded the maximum allowed size exception, item: " + record);
```

Comment:
So if "ValidationError" isn't the error code and error message isn't `Item size...`, this fails silently?